analytic_engine/src/instance/write.rs

```diff
@@ -517,6 +529,8 @@ impl Instance {
         table_data: &TableData,
         encoded_rows: Vec<ByteVec>,
     ) -> Result<SequenceNumber> {
+        let _timer = table_data.metrics.start_table_write_wal_timer();
```

Comment:
`write_table_row_group` fn also need a timer?

Reply:
Add a timer for the total write duration.